explorer/src/components/account/MetadataAttributesCard.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react'
+import { NFTData } from 'providers/accounts'
+import { LoadingCard } from 'components/common/LoadingCard'
+import { ErrorCard } from 'components/common/ErrorCard'
+
+interface Attribute {
+  trait_type: string
+  value: string
+}
+
+export function MetadataAttributesCard({ nftData }: { nftData: NFTData }) {
```

Comment:
nit: can you rename to `MetaplexNFTAttributesCard` since this is metaplex specific?

Reply:
This is a better name for understanding I agree, it has been renamed in this commit [chore: rename component and format](https://github.com/solana-labs/solana/pull/24580/commits/43ff215b19cd44c2e0a27ebe369c12b6d90100c4)